packages/identifier/src/reserved.ts

```diff
@@ -1128,18 +1131,37 @@ const slurs = [
   'squaws',
   'wetback',
   'wetbacks',
-  'yid',
   'yids',
 ]
 
 export const reservedSubdomains: Record<string, boolean> = [
   ...atpSpecific,
   ...commonlyReserved,
-  ...famousAccounts,
-  ...slurs,
 ].reduce((acc, cur) => {
   return {
     ...acc,
     [cur]: true,
   }
 }, {})
+
+export const dangerousUnallowedSubdomains = {
+  lowCheck: famousAccounts,
+  highCheck: slurs,
+}
+
+export const checkIfStringContainsSlurs = (
+  input: string,
+  unallowedSubdomains = dangerousUnallowedSubdomains,
+): boolean => {
+  const handleParts = input.toLowerCase().split('.')
+  const normalizedInput = input.toLowerCase().replace('.', '')
+
+  return (
+    !!unallowedSubdomains.lowCheck.find((unallowedString) =>
+      handleParts.includes(unallowedString),
+    ) ||
+    !!unallowedSubdomains.highCheck.find((unallowedString) =>
+      normalizedInput.toLowerCase().includes(unallowedString),
+    )
+  )
+}
```

Comment:
Minor feedback: This part could maybe be broken up to be clearer. It seems like this function is checking 2 different things?
* normalized strings that contain slurs
* subdomain parts w/ reserved names